server/auth.go

```diff
@@ -124,11 +124,16 @@ func (server *HTTPServer) Authentication(next http.Handler, noAuthURLs []string)
 			log.Error().Msg("org_id not found on top level in token structure (old format)")
 		}
 
-		if tk.Identity.AccountNumber == "" || tk.Identity.Internal.OrgID == 0 {
-			msg := fmt.Sprintf("error retrieving requester data from token. org_id [%v], account_number [%v], user data [%+v]",
-				tk.Identity.Internal.OrgID,
+		if tk.Identity.AccountNumber == "" || tk.Identity.AccountNumber == "0" {
```

Comment:
oh so "0" is special?

Reply:
they mentioned in can be missing altogether from the token, but I don't trust anyone so I added the "0" condition "just in case"..